client/src/elm/Pages/Prenatal/Activity/Utils.elm

```diff
@@ -3594,36 +3594,69 @@ resolveMedicationTreatmentFormInputsAndTasksCommon language currentDate setBoolI
                     else
                         ( [], [] )
             in
-            ( [ header
-              , viewQuestionLabel language Translate.TreatmentReviewQuestionStillTaking
```

Comment:
This is the only label that should change.
Everything else is code duplication.